src/main/java/se/sundsvall/supportmanagement/service/mapper/ErrandParameterMapper.java

```diff
@@ -11,6 +11,10 @@
 
 public final class ErrandParameterMapper {
 
+
+	private ErrandParameterMapper() {
+	}
```

Comment:
add a comment why this is empty
something like
		`// Intentionally empty`
		or
		`// To prevent instantiation`